### ISSUE TITLE
Missing/Wrong Aria-Roles giving Axe Violation.

### ISSUE THREAD

**Issue by saket2403** [cerner/terra-framework#945]
## Description
Certain components have no roles or wrong role-attribute combination assigned which are giving violations on the axe plugin.

## Error Report
` Certain ARIA roles must contain particular children`

`Elements must only use allowed ARIA attributes`

## Components

- Aggregator
- Collapsible Menu View
- Tabs
